aapp_runner/__init__.py

```diff
@@ -22,3 +22,6 @@
 
 """Initialise the aapp_runner package
 """
+
+from setuptools_scm import get_version
+version = get_version(root='..', relative_to=__file__)
```

Comment:
I think you should be able to do `from .version import version as __version__`.

Reply:
Ok, not sure I follow, what is wrong with the above?
I was looking here: https://pypi.org/project/setuptools-scm/

Reply:
Ok. I've never understood that function. I think it is supposed to be used in `setup.py` but then they give examples like the one you're using. If the package is installed then a `<pkg>/version.py` should exist so you can just import the module directly:

```
from .version import version as __version__
```

which would make it possible to do:

```
import aapp_runner
print(aapp_runner.__version__)
```

Reply:
Ah, I see, yes, that seem to work, and no need to make other imports!